### PR TITLE
Update Encrypted Data section in Security Considerations.

### PR DESCRIPTION
**Pull request by msporny**
Partial editorial cleanup to appendices tracked as issue #728.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/747.html" title="Last updated on May 29, 2021, 7:01 PM UTC (217bd25)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/747/435a304...217bd25.html" title="Last updated on May 29, 2021, 7:01 PM UTC (217bd25)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4574,20 +4574,24 @@ <h2>Immutability</h2>
...
+<a>DID documents</a> is not an appropriate means to protect personal data.
       </p>
       <p>
 Given the caveats above, if encrypted data is included in a <a>DID document</a>,
-implementers are advised to not encrypt with the public keys of entities that
-do not wish to be correlated with the <a>DID</a>.
+implementers are advised to not associate any correlatable information
+that could be used to infer a relationship between the encrypted data
+and an associated party. Examples of correlatable information include
+receiving party public keys, identifiers to digital assets known to be
```
Comment (TallTed) [inline index.html:4592]:
```suggestion
public keys of a receiving party, identifiers to digital assets known to be
```

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -4574,20 +4574,24 @@ <h2>Immutability</h2>
...
+and computing power. Implementers are advised to assume that any encrypted data
+placed in a <a>DID document</a> might eventually be made available in clear text
+to the same audience to which the encrypted data is available. This is
+particularly pertinent if the <a>DID document</a> is public.
       </p>
       <p>
-Encrypting all or parts of <a>DID documents</a> is <em>not</em> an appropriate means
-to protect data in the long term. Similarly, placing encrypted data in <a>DID
-documents</a> is not an appropriate means to include personal data.
+Encrypting all or parts of <a>DID documents</a> is <em>not</em> an appropriate
```
Comment (kdenhartog) [inline index.html:4583]:
```suggestion
Encrypting all or parts of a <a>DID document</a> is <em>not</em> an appropriate
```

File: index.html [hunk 3]
```diff
@@ -4574,20 +4574,24 @@ <h2>Immutability</h2>
...
+means to protect data in the long term. Similarly, placing encrypted data in
+<a>DID documents</a> is not an appropriate means to protect personal data.
```
Comment (kdenhartog) [inline index.html:4585]:
```suggestion
a <a>DID document</a> is not an appropriate means to protect personal data.
```

**Review by kdenhartog — APPROVED** [review]
minor editorial comments for consistent usage. LGTM

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**